src/components/Button/Button.module.css

```diff
@@ -31,13 +31,12 @@
 }
 
 .GitHub {
-  display: flex;
+  display: inline-flex;
+  align-items: center;
+  background: var(--bg-2);
   font-size: 16px;
-  align-items: baseline;
   color: var(--txt-1);
-  background: var(--bg-2);
   border-color: var(--bg-3);
-  justify-content: center;
```

Comment:
This feature should not be removed.

![image](https://user-images.githubusercontent.com/7966133/145763896-3352a550-bd44-4130-a88d-03cad5ae68ef.png)